wrapping/src/openmeeg.i

```diff
@@ -111,7 +111,7 @@ namespace std {
     %template(vector_vertex) vector<OpenMEEG::Vertex>;
     %template(vector_pvertex) vector<OpenMEEG::Vertex *>;
     %template(vector_triangle) vector<OpenMEEG::Triangle>;
-    %template(vector_mesh) vector<OpenMEEG::Mesh>;
+#    %template(vector_mesh) vector<OpenMEEG::Mesh>;
```

Comment:
cleanup? what did you have to change this?

Reply:
As I explained, swig does not know to wrap std::vector<X> when X has a deleted copy constructor. There are workarounds I need to investigate. But on the other end, I'm not sure we really need the Meshes. At least for now this was not tested at all....
The travis failure shows that one of the methods I removed was useful in the vtk load.
I will eventually restore that one at least to see if this solves the travis pb.